docker/lisk-core/docker-compose.core.yml

```diff
@@ -2,13 +2,16 @@ version: '3'
 services:
   lisk-core:
     image: lisk/core:3.0.0-beta.4
+    volumes:
+      - ./lisk-core/snapshots/:/tmp/snapshots
+      - ./lisk-core/entrypoint.sh:/entrypoint.sh
     ports:
       - 4000
       - 5001
     networks:
       - local_lisk_core
     restart: always
-    command: ['--network=devnet', '--api-ws', '--api-ws-port=5001', '--enable-forger-plugin', '--enable-http-api-plugin', '--http-api-plugin-port=4000', '--http-api-plugin-whitelist=172.16.0.0/12']
+    entrypoint: ['sh', '/entrypoint.sh']
```

Comment:
Does the script also work when you restart the container?

Reply:
I did not test this scenario. Let me verify and update.

Reply:
Updated the script to overwrite the blockchain data if it already exists.